whatrecord/bin/graph.py

```diff
@@ -0,0 +1,220 @@
+"""
+"whatrecord graph" is used to parse and generate a relationship graph of
+whatrecord-supported file types.
+"""
+
+import argparse
+import logging
+import re
+from typing import Dict, List, Optional
+
+# from ..access_security import AccessSecurityConfig
+from ..common import AnyPath, RecordInstance  # , FileFormat, IocMetadata
+from ..db import Database, LinterResults
+# from ..dbtemplate import TemplateSubstitution
+# from ..format import FormatContext
+# from ..gateway import PVList as GatewayPVList
+from ..graph import build_database_relations, graph_links
+# from ..macro import MacroContext
+from ..shell import LoadedIoc
+# from ..snl import SequencerProgram
+# from ..streamdevice import StreamProtocol
+from .parse import parse_from_cli_args
+
+logger = logging.getLogger(__name__)
+DESCRIPTION = __doc__
+
+
+def build_arg_parser(parser=None):
+    if parser is None:
+        parser = argparse.ArgumentParser()
+
+    parser.description = DESCRIPTION
+    parser.formatter_class = argparse.RawTextHelpFormatter
```

Comment:
It doesn't seem like your help strings are pre-formatted. What does using the RawTextHelpFormatter help you do?

Reply:
Mostly helps with keeping `DESCRIPTION` in tact, if I ever get around to making it a bit more verbose. Good catch in that it's unnecessary here :+1: (it's a copy/paste job of all of the other subcommand entrypoints 😁 )